exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -557,8 +557,7 @@ private ExecConstants() {
   public static final LongValidator CODE_GEN_EXP_IN_METHOD_SIZE_VALIDATOR = new LongValidator(CODE_GEN_EXP_IN_METHOD_SIZE);
 
   /**
-   * Timeout for create prepare statement request. If the request exceeds this timeout, then request is timed out.
-   * Default value is 10mins.
+   * Default value is 30 seconds.
```

Comment:
Maybe we should remove comment about the default as well? Actual default value can be found in drill-module.conf and if it changes, we won't need to modify it here.